apps/src/templates/projects/ProjectCardGrid.story.jsx

```diff
@@ -78,61 +78,38 @@ function generateFakePersonalProjects() {
   return personalProjects;
 }
 
-const createProjectsStore = function() {
-  return createStore(combineReducers({projects}));
+const createProjectsStore = function(galleryType) {
+  return reduxStore({projects}, {projects: {selectedGallery: galleryType}});
 };
 
-export default storybook => {
-  storybook.storiesOf('Projects/ProjectCardGrid', module).addStoryTable([
-    {
-      name: 'Public Gallery with student info',
-      description:
-        'Public gallery, with shortened student names and student age ranges.',
-      story: () => {
-        const store = createProjectsStore();
-        store.dispatch(selectGallery(Galleries.PUBlIC));
-        return (
-          <Provider store={store}>
-            <ProjectCardGrid
-              projectLists={generateFakePublicProjectsWithStudentInfo()}
-              galleryType="public"
-            />
-          </Provider>
-        );
-      }
-    },
-    {
-      name: 'Public Gallery without student info',
-      description: 'Public gallery, without student name and age.',
-      story: () => {
-        const store = createProjectsStore();
-        store.dispatch(selectGallery(Galleries.PUBlIC));
-        return (
-          <Provider store={store}>
-            <ProjectCardGrid
-              projectLists={generateFakePublicProjectsWithoutStudentInfo()}
-              galleryType="public"
-            />
-          </Provider>
-        );
-      }
-    },
-    {
-      name: 'Personal Gallery',
-      description:
-        'Personal gallery, showing full names and the "quick action" dropdowns',
-      story: () => {
-        const store = createProjectsStore();
-        store.dispatch(selectGallery(Galleries.PUBlIC));
-        return (
-          <Provider store={store}>
-            <ProjectCardGrid
-              projectLists={generateFakePersonalProjects()}
-              galleryType="personal"
-            />
-          </Provider>
-        );
-      }
-    }
-  ]);
+export default {
+  title: 'ProjectCardGrid',
+  component: ProjectCardGrid
+};
+
+const Template = (galleryType, args) => (
+  <Provider store={createProjectsStore(galleryType)}>
+    <ProjectCardGrid {...args} />
+  </Provider>
+);
+
+export const PublicGalleryWithStudentInfo = Template.bind(Galleries.PUBLIC, {});
```

Comment:
I'm confused by bind syntax here, help me out -- my understanding is calling bind with a blank object as the first arg (like we usually do in this new storybook world) creates a copy of the function, but you're providing a string? How does this work?

Reply:
I think you are right, I misread [this doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind), you can provide args but the first one should be `this`, I'll fix this up

Reply:
I think it still worked in this case because we don't use `this` and the arg I am providing may not be necessary

Reply:
Got it, thanks for clarifying!